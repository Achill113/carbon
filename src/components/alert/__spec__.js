import React from 'react';
import TestUtils from 'react/lib/ReactTestUtils';
import Alert from './index';

describe('Alert', () => {
  it('failing test demo', () => {
    expect(1).toEqual(2);
  });
});
