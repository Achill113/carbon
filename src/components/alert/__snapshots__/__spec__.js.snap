// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert include correct component, element and role data tags 1`] = `
<div
  className="carbon-dialog"
  data-component="alert"
  data-element="bar"
  data-role="baz"
  data-state="closed"
>
  <CSSTransitionGroup
    component="div"
    transitionAppear={false}
    transitionEnter={true}
    transitionEnterTimeout={500}
    transitionLeave={true}
    transitionLeaveTimeout={500}
    transitionName="modal"
  />
  <CSSTransitionGroup
    component="div"
    transitionAppear={false}
    transitionEnter={true}
    transitionEnterTimeout={500}
    transitionLeave={true}
    transitionLeaveTimeout={500}
    transitionName="modal-background"
  />
</div>
`;
